FV3/coarse_graining/coarse_graining.F90

```diff
@@ -300,6 +309,56 @@ subroutine weighted_block_edge_average_y_3d_field_2d_weights(weights, fine, coar
     enddo
   end subroutine weighted_block_edge_average_y_3d_field_2d_weights
 
+  subroutine vertically_remap_field(phalf_in, field, phalf_out, ptop, field_out)
```

Comment:
Does vertical remapping here mean the same thing as it does as used in the code grabbed, e.g. that is mapping lagraingian coordinates to a eulerian grid to prevent excessive vertical distortion?

Reply:
To the extent that both rely on interpolating from one set of pressure levels to another, it means the same thing.  Here we are using `mappm` as a convenient way to interpolate a field from pressure levels on the fine grid to pressure levels on the coarse grid.